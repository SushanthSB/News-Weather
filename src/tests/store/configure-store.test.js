import renderer from 'react-test-renderer';
import Store from '../../store/configure-store';

describe('Reducers()',() => {
	it('Should not be modified',() => {
		expect(Store).toMatchSnapshot();
	})
})