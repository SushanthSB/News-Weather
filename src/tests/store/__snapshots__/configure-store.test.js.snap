// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reducers() Should not be modified 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;
